vere/main.c

```diff
@@ -325,20 +317,18 @@ _main_getopt(c3_i argc, c3_c** argv)
     return c3n;
   }
   else if ( u3_Host.ops_u.nuu == c3y
-           && u3_Host.ops_u.url_c == 0 ) {
-           // XX find a way to re-enable -s (auto-pill)
-           // && u3_Host.ops_u.git == c3n ) {
+           && u3_Host.ops_u.url_c == 0
+           && u3_Host.ops_u.git == c3n ) {
     u3_Host.ops_u.url_c =
       "https://bootstrap.urbit.org/urbit-" URBIT_VERSION ".pill";
   }
-  // XX find a way to re-enable -A (fastboot)
-  // else if ( u3_Host.ops_u.nuu == c3y
-  //          && u3_Host.ops_u.url_c == 0
-  //          && u3_Host.ops_u.arv_c == 0 ) {
+  else if ( u3_Host.ops_u.nuu == c3y
+           && u3_Host.ops_u.url_c == 0
```

Comment:
-u with -A should be fine shouldn't it? I think this should check if ops_u.git == c3y

Reply:
Yeah, that should too. This validation is ported straight from master, but that doesn't mean it shouldn't be improved.

Reply:
Oh, it's an `else if`, so it's already conditioned on `ops_u.git`. I'm not completely sure this is right, but the argument validation logic really needs its own review pass. I'm inclined to leave this as is for now.